src/Windows/Avalonia.Win32/WindowImpl.AppWndProc.cs

```diff
@@ -27,9 +24,9 @@ public partial class WindowImpl
         protected virtual unsafe IntPtr AppWndProc(IntPtr hWnd, uint msg, IntPtr wParam, IntPtr lParam)
         {
             const double wheelDelta = 120.0;
-            const long UiaRootObjectId = -25;
+            const long uiaRootObjectId = -25;
```

Comment:
Why was it renamed to the lower case?

Reply:
For consistency with the variable above. I had a quick look at the codebase and the majority of local constants seem to follow camel case. It would be nice to have rules for all the naming conventions in the `.editorconfig` file!

Anyway, I apologize for style changes unrelated to nullability. I have the habit of quickly cleaning warnings in files I modify: add `readonly`, fix naming convention... I do that all the time, almost automatically, with projects I'm working on. I should be more careful next times and only keep the PR changes to the minimum. Let me know if you want me to reverse that.